README.md

```diff
@@ -57,9 +60,33 @@ unless real values don't help users know what to change.
 
 ```hcl
 
+module "watsonx_assistant" {
+  source            = "terraform-ibm-modules/watsonx-assistant/ibm"
+  region            = "us-south"
+  assistant_name    = "example-wx-assistant-basic"
+  plan              = "lite"
+  resource_group_id = module.resource_group.resource_group_id
+}
+
 ```
 
-### Required access policies
+### Required IAM access policies
+
+You need the following permissions to run this module:
+
+* Account Management
+  * **Resource Group**
+        - `Administrator` role
```

Comment:
why do you need `Administrator` role on resource group? The module doesn't create a resource. So you would only need Viewer access on the specific resource group

Reply:
You are right. Sorry for the oversight. I have made the change.